lib/Subscribers/ImportExport/Import/Import.php

```diff
@@ -297,14 +298,20 @@ private function setSubscriptionStatusToSubscribed($subscribers_data) {
     return $subscribers_data;
   }
 
-  function setSource($subscribers_data) {
+  private function setSourceAndLinkToken($subscribers_data) {
```

Comment:
Don't you think it would be clearer and easier to understand if there were two methods? One for setting the source and second for the token?

Reply:
You are right two methods will be nicer.